apps/main-landing/src/components/products-section/components/creators-section/creators-section-actions.tsx

```diff
@@ -0,0 +1,16 @@
+import { Button } from '@idriss-xyz/ui/button';
+export const CreatorsSectionActions = () => {
+  return (
+    <Button
+      intent="secondary"
+      size="large"
+      className="w-full text-button2 md:w-fit lg:w-fit lg:text-button1"
```

Comment:
why do we override text-size? It should be set correctly already in design system